app/presenters/organisations/featured_news_presenter.rb

```diff
@@ -0,0 +1,48 @@
+module Organisations
+  class FeaturedNewsPresenter
+    attr_reader :org
+
+    def initialize(organisation)
+      @org = organisation
+    end
+
+    def first_featured_news
+      main_story = featured_news([org.ordered_featured_documents.first])
```

Comment:
If we add the `[0]` to the first line, we no longer need the second line at all. E.g:

```
main_story = featured_news([org.ordered_featured_documents.first])[0]
main_story[:large] = true
main_story
```